.github/workflows/gh-pages.yml

```
@@ -0,0 +1,36 @@
+name: Publish Storybook
+on:
+  push:
+    branches:
+      - develop
+      - EGRC-275
+
+jobs:
+  publish:
+    name: Publish Storybook to GitHub Pages
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

```suggestion
        with:
          fetch-depth: 0
```

We actually don't need the whole history here; sorry for suggesting we copy/paste from the other workflow.